charts/agent/templates/_helpers.tpl

```diff
@@ -143,7 +143,7 @@ Return the default only if the value is not defined in sysdig.settings.<agent_se
 */}}
 {{- define "get_if_not_in_settings" -}}
 {{- if not (hasKey .root.Values.sysdig.settings .setting) -}}
-  {{- if .default -}}
+  {{- if and (ne .default nil) (printf "%s" .default) -}}
```

Comment:
will things be printed here??

Reply:
it's more like sprintf, it'll all be contained in that temporary expression. but the old helm 3.4 doesn't like this approach anyway so going to try something different